test/run-windows-k8s-integration.sh

```diff
@@ -9,9 +9,8 @@
 set -o nounset
 set -o errexit
 
-readonly GCE_PD_DO_DRIVER_BUILD=false
 readonly PKGDIR=${GOPATH}/src/sigs.k8s.io/gcp-compute-persistent-disk-csi-driver
-readonly LOCAL_K8S_DIR=${GOPATH}/src/k8s.io/kubernetes
+readonly k8s_dir=${PKGDIR}/kubernetes
```

Comment:
`PKGDIR` points to the PD CSI driver repo

Reply:
right, in prow test, kuberentes will be installed under current dir which is the pd csi driver repo. in the test, kubetest needs to be in kubernetes dir.

Reply:
I think we don't need this variable anymore?

Reply:
removed it. thanks!